distributions/replication-dominion/src/main/filtered-resources/features.xml

```diff
@@ -20,9 +20,9 @@
     <repository>mvn:org.codice.test/dominion-pax-exam-feature/${codice-test.version}/xml/features
     </repository>
     <repository>mvn:ddf.lib/dominion-commons/${ddf.version}/xml/features</repository>
-    <repository>mvn:org.codice.ddf/ddf-dominion/${ddf.version}/xml/features</repository>
+    <repository>mvn:org.codice.ddf/ddf-dominion/${dominion.version}/xml/features</repository>
 
-    <feature name="ddf-dominion" version="${project.version}">
+    <feature name="dominion" version="${project.version}">
```

Comment:
✏️ Not sure we should be renaming this one since it is the DDF extensions to dominion

Reply:
changed the name back to ddf-dominion